### ISSUE TITLE
Undo/Redo Functionality

### ISSUE THREAD

**Issue by kundana29** [vishanurag/Canvas-Editor#61]
Mistakes happen frequently when designing, and having to clear everything because of a small error is frustrating.Adding basic undo and redo buttons to allow users to go back a step or restore a previous state with ease.
please assign this issue to me

**Comment by vishanurag** [comment]
There are already undo redo buttons present in the webpage. But they are not functioning well. Can you please make the functionalities of these buttons well???

**Comment by kundana29** [comment]
yes i can . please asiign this to me under gssoc-ext anf hactoberfeast'24
